version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "0.22.0"
+version in ThisBuild := "0.23.0"
```

Comment:
The version should not be modified manually, it is automatically increased on the release process, so we need to revert this change :)

Reply:
Actually never mind this one, I can solve it via the conflict resolver of Github :)